WordPress/Sniffs/Theme/NoDeregisterCoreScriptSniff.php

```diff
@@ -6,7 +6,16 @@
  * @package  PHP_CodeSniffer
  * @author   Simon Prosser <pross@pross.org.uk>
  */
-class WordPress_Sniffs_Theme_NoDeregisterCoreScriptSniff extends WordPress_Sniffs_Functions_FunctionRestrictionsSniff {
+class WordPress_Sniffs_Theme_NoDeregisterCoreScriptSniff implements PHP_CodeSniffer_Sniff {
+
+	/**
+	 * Core scripts to sniff for.
+	 *
+	 * @var array
+	 */
+	public $core_scripts = array(
```

Comment:
@Pross Just wondering why you made this a `public` property not a `private` or `protected` one ? Public properties can be influenced via the `phpcs.xml` file, `private` and `protected` properties can not.

See: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml (slightly over half way down the page)